src/explore-education-statistics-common/src/components/Banner.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from '@common/components/Banner.module.scss';
+import useMounted from '@common/hooks/useMounted';
+import ButtonText from '@common/components/ButtonText';
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  children: ReactNode;
+  showBanner: boolean;
+  wide?: boolean;
+  onClose: () => void;
```

Comment:
We might want to make this optional as we might need the banner to always be shown in the future?

The relevant UI would also need to not render if this isn't set